corehq/apps/commtrack/templates/commtrack/manage/rebuild_stock_state.html

```diff
@@ -3,39 +3,78 @@
 {% load hq_shared_tags %}
 
 {% block main_column %}
-    <table class="table">
+    <table class="table table-bordered table-striped">
+    {% for key, actions, case_doc_info in actions_by_stock_state_key %}
+    {% for action_type, action in actions %}
+        {% with action.stock_transaction as stock_transaction %}
+        {% if forloop.first %}
+        <tr>
+            <td colspan="6" style="text-align: center">
+                <strong>{{ case_doc_info|pretty_doc_info }}</strong>
+                {{ stock_transaction.section_id }}
+                for <em>{{ stock_transaction.sql_product.name }}</em>
+                {% if request.user.is_superuser and request|toggle_enabled:'SUPPORT' %}
+                <form method="post" action="" class="pull-right">
+                    <input type="hidden" name="case_id" value="{{ key.case_id }}">
+                    <input type="hidden" name="section_id" value="{{ key.section_id }}">
+                    <input type="hidden" name="product_id" value="{{ key.product_id }}">
+                    <input type="submit" class="btn btn-primary" value="Rebuild"/>
+                </form>
+                {% endif %}
+            </td>
+        </tr>
+        <tr>
+            <th colspan="2"></th>
+            <th colspan="2">Current Values</th>
+            <th colspan="2">Rebuilt Values</th>
```

Comment:
i know we expect this page to be internal, but it doesn't hurt to throw `trans` tags on these